flocker/provision/_aws.py

```diff
@@ -58,6 +58,13 @@
 INSTANCE_TIMEOUT = 300
 
 
+class FailedToRun(Exception):
+    """
+    Raised if a pending AWS failed to become running.
+    See: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-lifecycle.html
+    """     # noqa
```

Comment:
Blech.  `noqa` really bothers me.  It might be okay if all it did was suppress "line too long" warnings.  Instead, it suppresses any and all warnings. :(

I don't have a good suggestion about what to do instead, though.  It just seems like the cure is worse than the disease here.